text/0001-mansona-proposed-unique-id-helper.md

```diff
@@ -9,8 +9,8 @@ teams:
   - framework
   - learning
 proposal-pr: https://github.com/emberjs/rfcs/pull/1
-tracking-project: https://github.com/mansona/rfcs/issues/2
-stage: Accepted
+tracking-project: https://github.com/emberjs/rfcs/pull/3
```

Comment:
Discussions with @jenweber: it's unlikely that we should force people to change this that often. We might be best to just leave one tracking project for the entire length of the RFC